packages/debian/entrypoint.sh

```diff
@@ -36,10 +36,27 @@ rpm2deb() {
 }
 
 
+buildrepo() {
+    set -x
+    mkdir -p /var/www/debian/{conf,incoming}
+    # We assume that all downloaded packages are mounted in /metalk8s-deb/
+    cp /metalk8s-deb/distributions /var/www/debian/conf
+    # Mount `distributions` file in  /var/www/debian/conf
```

Comment:
It's not a mount, you're copying it. IMO, the comment can be removed.

Reply:
The file `distributions` is created before the repository build, it should be mounted in the container then, paste in the folder `/var/www/debian/conf`.
This comment is just a reminder for the next PR, integrate this script in the buildchain (it will be deleted after)